### PR TITLE
V3

### PR DESCRIPTION
**Pull request by ericawright**
Version 3.
- updated version and changed slugs
- updated to latest multipreffer (removes uninstall-on-disable) 
- please double check the strict list pref for me, I see it is different from last time. I'm no longer sure which is correct, this was taken from:
https://mozilla.slack.com/archives/CAPE8A1K2/p1558560115041800?thread_ts=1558560115.041800

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "cookie-restrictions-strict-list-study",
+  "name": "cookie-restrictions-strict-list",
```
Comment (nhnt11) [inline package.json]: Out of curiosity, why this change?

File: src/variations.json [hunk 2]
```diff
@@ -2,15 +2,13 @@
...
-      "urlclassifier.trackingAnnotationTable": "test-track-simple,base-track-digest256",
-      "network.cookie.cookieBehavior": 0
     }
   },
 
   "test": {
     "weight": 1,
     "prefs": {
-      "urlclassifier.trackingAnnotationTable": "test-track-simple,base-track-digest256,content-track-digest256",
+      "urlclassifier.trackingAnnotationTable": "test-track-simple,ads-track-digest256,social-track-digest256,analytics-track-digest256,content-track-digest256",
```
Comment (nhnt11) [inline src/variations.json]: Could you follow up with Tanvi and/or Baku about this, please?

File: package.json (hunk 1 above)
Comment (ericawright) [reply package.json]: mythmon requested a new extension id so we can classify it as a different experiment. I don't know the details as to why.

File: src/variations.json (hunk 2 above)
Comment (ericawright) [reply src/variations.json]: This has been resolved, it turns out these two values are different ways of writing the same thing. And all versions of Firefox can handle both properly.
